Ownership/TC - LTC/London TC WELIN.txt

```diff
@@ -52,3 +70,19 @@ ALTOWNER:Observing LON_C_CTR:TCGS:WELIN:TCM:LML:LME:LGS:LM:O:LMEN:LC:LSC:L
 ALTOWNER:Observing LON_SC_CTR:TCGS:WELIN:TCM:LML:LME:LM:LC:LGS:O:LMEN:LSC:L
 ALTOWNER:Observing London FIR:TCGS:LGS:O:LMEN:WELIN:TCM:LML:LME:LM:LC:LSC:L
 BORDER:LCE2b:LTC55:LTC2
+
+SECTOR:London TC WELIN 7 (000-215):00000:21500
```

Comment:
moving away from DB as the base? see TC WELIN 4 for example...

Reply:
Fixed!